odl/discr/lp_discr.py

```diff
@@ -626,9 +818,12 @@ def uniform_discr_fromspace(fspace, nsamples, exponent=2.0, interp='nearest',
             'nearest' : use nearest-neighbor interpolation (default)
 
             'linear' : use linear interpolation (not implemented)
+
     impl : {'numpy', 'cuda'}
         Implementation of the data storage arrays
 
+    Other Parameters
```

Comment:
Yes. From the numpydoc doc:

    7. Other Parameters

    An optional section used to describe infrequently used parameters.
    It should only be used if a function has a large number of keyword parameters,
    to prevent cluttering the Parameters section.

Perhaps an over-use here, but sometimes it may be worth separating the most important parameters from the advanced, extra ones. AFAIK they end up further down in the Sphinx doc also.